include/buildpack.bash

```diff
@@ -56,6 +79,7 @@ buildpack-setup() {
 		"$app_path" \
 		"$build_path" \
 		"$cache_path" \
+		"$env_path" \
```

Comment:
The embjerjs buildpack attempts to write to `$env_path` [here](https://github.com/heroku/heroku-buildpack-emberjs/blob/4d0298b85301458c86a7a1a62d3d73feb92b20ed/buildpack/mrblib/buildpack/commands/compile.rb#L29-L31)

It would seem no other buildpack does this. Without this particular change, the emberjs buildpack fails thusly:
```
-----> Setting NPM_CONFIG_PRODUCTION to false to install ember-cli toolchain
RuntimeError: open
```